SummaryJavadoc/Example2/list-of-projects.properties

```diff
@@ -38,7 +38,9 @@ jOOL|git|https://github.com/jOOQ/jOOL|version-0.9.7||
 RxJava|git|https://github.com/ReactiveX/RxJava|v1.0.9||
 Vavr|git|https://github.com/vavr-io/vavr|v0.9.0||
 
-# Few projects with excludes to decrease a number of checked files (usefull for some checks which specify overly strong code style policy)
+# For special checks only
+
+# Few projects with excludes to decrease a number of checked files (useful for some checks which specify overly strong code style policy)
```

Comment:
lets keep in below only 
```
# custom javadoc tags
nbia-dcm4che-tools|git|https://github.com/thprakash/nbia-dcm4che-tools|c3591e6f0f84827586db25abded6708e5386ef1a||
```

as it is about javadocs, so  useful for javadoc checks

Reply:
this will be good example that some Check will need very special set of projects.
No reason to run regression testing of SummaryJavadocCheck on project that is not library so thy do not care about writing javadoc at all.

Reply:
okay, i assume i have commented eveyrthing except custom javadoc tags. please let me know if it is done right. also, i realise we missed all-in-one-examples for javadoc.

Reply:
> realise we missed all-in-one-examples for javadoc

please send in separate PR.